sql/core/src/main/scala/org/apache/spark/sql/execution/command/createDataSourceTables.scala

```diff
@@ -131,6 +135,8 @@ case class CreateDataSourceTableAsSelectCommand(
     val tableName = table.identifier.unquotedString
```

Comment:
Just a suggestion. Since we already added the database into the identifier, we can use fully qualified names in the error messages. It will be much easier for users to know which table it is. That means, we can change this line to
```Scala
val tableName = tableIdentWithDB.identifier.unquotedString
```

Below is an example of DB2. The error message shows the fully qualified name.
```
db2 => drop table ta
DB21034E  The command was processed as an SQL statement because it was not a 
valid Command Line Processor command.  During SQL processing it returned:
SQL0204N  "DB2INST1.TA" is an undefined name.  SQLSTATE=42704
```